docs/sources/flow/reference/components/loki.write.md

```diff
@@ -101,6 +102,11 @@ Endpoints can be named for easier identification in debug metrics by using the
 `name` argument. If the `name` argument isn't provided, a name is generated
 based on a hash of the endpoint settings.
 
+The `retry_on_http_429` argument specifies whether `HTTP 429` status code
+responses should be treated as recoverable errors; other `HTTP 4xx` status code
+responses are never considered recoverable errors. When `retry_on_http_429` is
+enabled, the retry mechanism will be governed by the specified `backoff` configuration.
```

Comment:
What is the `backoff` configuration? We don't have a block or argument with this name ;) 
```suggestion
enabled, the retry mechanism will be governed by the backoff configuration specified through `min_backoff_period`, `max_backoff_period ` and `max_backoff_retries` attributes.
```

Reply:
True, thought it was clear enough with the different backoff attributes in the table above but that makes it clearer